spec/busted-ci-helper.lua

```diff
@@ -1,10 +1,37 @@
 -- busted-ci-helper.lua
+local busted = require("busted")
 
-local busted = require 'busted'
-local cjson = require 'cjson'
-local socket_unix = require 'socket.unix'
+local function cleanup()
+  local prefix = assert(require("kong.conf_loader")(os.getenv("KONG_SPEC_TEST_CONF_PATH") or "spec/kong_tests.conf")).prefix
```

Comment:
It seems that only one prefix directory is being deleted here. However, if there are tests like `spec-ee/02-integration/03-vitals/08-hybrid_spec.lua` that create multiple prefix paths, how can we clear all of them?